src/lib/Scenes/Artwork/Components/OtherWorks/ArtworkContexts/ArtworkContextAuction.tsx

```diff
@@ -0,0 +1,37 @@
+import { ArtworkContextAuction_artwork } from "__generated__/ArtworkContextAuction_artwork.graphql"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+
+import { ContextSectionsContainer } from "../ContextSectionsContainer"
+import { ArtistArtworkGrid, AuctionArtworkGrid, RelatedArtworkGrid } from "./ArtworkGrids"
+
+export const ArtworkContextAuctionFragmentContainer = createFragmentContainer<{
+  artwork: ArtworkContextAuction_artwork
+  isClosed: boolean
+}>(
+  props => {
+    if (props.isClosed) {
+      return (
+        <ContextSectionsContainer>
+          <ArtistArtworkGrid artwork={props.artwork} />
+          <RelatedArtworkGrid artwork={props.artwork} />
+        </ContextSectionsContainer>
+      )
+    } else {
+      return (
+        <ContextSectionsContainer>
+          <AuctionArtworkGrid artwork={props.artwork} />
+        </ContextSectionsContainer>
+      )
+    }
+  },
+  {
+    artwork: graphql`
+      fragment ArtworkContextAuction_artwork on Artwork @argumentDefinitions(excludeArtworkIDs: { type: "[String!]" }) {
```

Comment:
In the web implementation, we conditionally fetch for these depending on the value of `isClosed`, but that's easier because these are all rendered below the fold and fetched client-side.

We can do that here if we want, but I also want to verify with Sam whether or not it's desired-- we may be able to remove some complexity here.